### ISSUE TITLE
Broken auto completion in vscode on instance object

### ISSUE THREAD

**Issue by moritzraho** [adobe/aio-lib-state#25]
```
const state = await stateLib.init() // => autocompletes
state. // => doesn't autocomplete
```

vscode's intellisense uses typescript types which seem to be compatible/inferred from our JSDoc. But there might be some sort of conflict.

**Comment by moritzraho** [comment]
relates to https://github.com/adobe/aio-lib-files/issues/24

**Comment by shazron** [comment]
fixed in #35
